modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java

```diff
@@ -368,6 +374,16 @@ public void execute() throws MojoExecutionException {
                 }
             }
 
+            if (skipIfSourceIsOlderThanOutput) {
+                if (inputSpecFile.exists() && output.exists()) {
+                    if (inputSpecFile.lastModified() <= output.lastModified()) {
```

Comment:
`inputSpecFile` here would reference an OpenAPI document and `output` refers to the output directory.

Is the intention of `skipIfSourceIsOlderThanOutput` to only regenerate if someone has modified the spec document since the last regeneration? If this is the case, I would suggest renaming the property to avoid confusion… something like `skipIfSpecIsUnchanged` might be clearer to those reading the properties list. Even after reading the issue and scanning the PR's changes, I thought this was doing something similar to `skipOvewrite`.

Although… my suggested `skipIfSpecIsUnchanged` might suggest to users that we're storing the atime/mtime of the spec (which we could definitely persisted to `{output}/.openapi-generator/SOME_FILENAME`).